css/build.css

```diff
@@ -42,11 +42,13 @@
 }
 
 .list.list-thumb-l ul > li {
-	list-style: none;
+	align-items: center;
 	border-bottom: 1px solid rgba(51,51,51,0.2);
+	display: flex;
+	list-style: none;
+	min-height: 71px;
```

Comment:
Why `71`?

Reply:
I added only display: flex; and align-items: center.  Min-height: 71px was before. I just rearranged all properties alphabetically.

Reply:
✅